src/tag/README.md

```diff
@@ -2,15 +2,20 @@
 
 The `vscode-tag` component is used to label an item, attract attention, and/or flag status.
 
-## Attributes
-
-None
+![Tag hero](/docs/assets/images/tag-hero.png)
 
 ## Usage
 
-A `vscode-tag` should only contain text and will be automatically be converted uppercase to follow the conventions of the Visual Studio Code design language.
+| ❌ Don't                                                                                                                                                  | ✅ Do                                                              |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
+| ![A tag incorrectly displaying numbers](/docs/assets/images/tag-dont-1.png)                                                                               | ![A tag using a text label](/docs/assets/images/tag-do-1.png)      |
+| Don't use tags to display numbers. If a component that highlights an item with a number is desired, see the [vscode-badge](../badge/README.md) component. | Use tags as text labels to indicate purpose or status for content. |
```

Comment:
Why the dash in "FEATURE-REQUEST"?

<img width="441" alt="Screen Shot 2022-02-04 at 11 38 49 AM" src="https://user-images.githubusercontent.com/39639992/152592566-d54b10b0-63b8-4de9-8314-25cb6a4dc0f5.png">